text/0000-stack-buildpacks.md

```diff
@@ -0,0 +1,606 @@
+  ...
+  [-stack-buildpacks <stack-buildpacks>] \
+  [-stack-group <group>]
+```
+
+##### Inputs
+| Input                 | Environment Variable        | Default Value           | Description
+|-----------------------|-----------------------------|-------------------------|----------------------
+| ...                   |                             |                         |
+| `<stack-buildpacks>`  | `CNB_STACK_BUILDPACKS_DIR`  | `/cnb/stack/buildpacks` | Path to stack buildpacks directory (see [Buildpacks Directory Layout]
+| `<stack-group>`       | `CNB_STACK_GROUP_PATH`      | `./stack-group.toml`    | Path to output group definition(#buildpacks-directory-layout))
+
+##### Outputs
+| Output             | Description
+|--------------------|----------------------------------------------
+| `<stack-group>`    | Detected stack buildpack group  (see [`group.toml`](#grouptoml-toml))
+
+#### `builder`
+The platform MUST execute `builder` in the **build environment**
+
+If the `<stack-group>` IS provided, the `builder` MUST be run with root privileges. If the the `<stack-group>` IS NOT provided, the `builder` MUST NOT be run with root privileges.
+
+Usage:
+```
+/cnb/lifecycle/builder \
+  ...
+  [-stack-buildpacks <stack-buildpacks>] \
+  [-stack-group <stack-group>]
+```
+
+##### Inputs
+| Input                 | Environment Variable        | Default Value           | Description
+|-----------------------|-----------------------------|-------------------------|----------------------
+| ...                   |                             |                         |
+| `<stack-buildpacks>`  | `CNB_STACK_BUILDPACKS_DIR`  | `/cnb/stack/buildpacks` | Path to stack buildpacks directory (see [Buildpacks Directory Layout]
+| `<stack-group>`       | `CNB_STACK_GROUP_PATH`      | `./stack-group.toml`    | Path to output group definition(#buildpacks-directory-layout))
+
+##### Outputs
+| Output                                     | Description
+|--------------------------------------------|----------------------------------------------
+| ...                                        |
+| `<layers>/<buildpack ID>.tgz`              | Layer snapshot (see [Buildpack Interface Specfication](buildpack.md)
+
+#### `exporter`
+Usage:
+```
+/cnb/lifecycle/exporter \
+  [-stack-group <stack-group>] \
+  ...
+```
+
+##### Inputs
+| Input               | Environment Variable  | Default Value       | Description
+|---------------------|-----------------------|---------------------|---------------------------------------
+| ...                 |                             |                           |
+| `<stack-group>`     | `CNB_STACK_GROUP_PATH`      | `./stack-group.toml`      | Path to stack-group file (see [`group.toml`](#grouptoml-toml))
+
+
+#### `rebaser`
+Usage:
+```
+/cnb/lifecycle/rebaser \
+  [-build-image <build-image>] \
+  ...
+```
+
+##### Inputs
+| Input               | Environment Variable  | Default Value          | Description
```

Comment:
Do we need a cache as an input to rebaser?

Reply:
Addressed in 43a7d1d